airflow/models/abstractoperator.py

```diff
@@ -345,13 +349,22 @@ def _do_render_template_fields(
                 )
             if not value:
                 continue
-            rendered_content = self.render_template(
-                value,
-                context,
-                jinja_env,
-                seen_oids,
-            )
-            setattr(parent, attr_name, rendered_content)
+            try:
+                rendered_content = self.render_template(
+                    value,
+                    context,
+                    jinja_env,
+                    seen_oids,
+                )
+            except Exception:
+                logger.exception(
+                    f"Exception rendering Jinja template for in "
+                    f"task '{self.task_id}', field "
+                    f"'{attr_name}'. Template: {value!r}"
+                )
```

Comment:
I think `self.log.exception` is better here; `baseoperator` needed an additional `logger` for free functions outside a class, but this inherits `LoggingMixin` and does not need it.

Reply:
Done!